content/topics/Analyze/causal-inference/matching/approximate-matching.md

```diff
@@ -0,0 +1,246 @@
+draft: false
+weight: 2
+author: "Valerie Vossen"
+aliases:
+  - /approximate-matching
+  - /propensity-score-matching
+---
+
+## Overview
+
+[Here](/matching) we introduced matching where we specifically discussed exact matching, the method in which individuals are paired on one or more *identical* characteristics. 
+
+Now we will continue with approximate matching, where pairs are made on *similar but not identical* characteristics. It is again a way to adjust for differences when treatment is non-random. Important to mention is that this method is only relevant for selection on observables, which means you should be able to identify the confounders (and have data on them!) that are biasing the effect. We focus on Propensity Score Matching specifically and provide a code example in R. 
+
+## Curse of dimensionality
+
+[Exact matching](/matching) requires an untreated unit with the precise similar value on each of the observed characteristics to be a good counterfactual for the treated units. The curse of dimensionality means that when there is a large number of variables you are matching treated and untreated units, or when some variables are continuous, there might be no appropriate counterfactual left (that has all the same values on all these covariates). This can be an issue in finite samples, where you have too many covariates to match and not enough data.
+
+Approximate matching offers a solution; it reduces the dimensionality by defining a distance metric on characteristic X and pair units based on this distance rather than the value of X. As the sample gets larger, approximate matching method results will all tend towards exact matching. 
+
+
+## Approximate matching methods
+
+{{% summary %}}
+
+Common methods of approximate matching include: 
+
+- *Nearest-neighbour matching*: Pairs each treated unit with the "nearest" untreated unit based on defined characteristics. 
+
+- *Kernel matching*: Estimates the counterfactual outcome of the treated unit by averaging outcomes of nearby control units. 
+
+- *Propensity score matching*: Uses the propensity score, representing the probability of treatment assignment given observed covariates.
+
+The guide of [Glen Waddel](https://pages.uoregon.edu/waddell/metrics/matching.html) is a good resource that discusses these types in more detail.
+
+{{% /summary %}}
+
+As the sample size increases, different matching estimators will yield similar results. However, their performance varies in finite samples. The choice depends on your data. [Glen Waddel](https://pages.uoregon.edu/waddell/metrics/matching.html) also gives a broader discussion on this and some points to consider when choosing the method. 
+
+
+## Propensity score matching
+
+Propensity score matching addresses the curse of dimensionality by matching on the "propensity score", representing the probability of being assigned to the treatment group, conditional on the particular covariates $X$:
```

Comment:
You could add that when it is true that conditional independence holds conditional on X, then it is also the case that conditional independence also holds conditional on the propensity score. This is a famous result by Rosenbaum and Rubin (1983). It is useful because X can be high-dimensional, while the propensity score is one-dimensional. So, we can match on something that is more likely to have common support and it will be easier to have good (closer) matches.